tests/integration/targets/docker_image/tasks/tests/options.yml

```diff
@@ -3,6 +3,9 @@
 # GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
+- name: Include docker_image test variables
+  include_vars: '../../vars/main.yml'
+
```

Comment:
```suggestion
```
`vars/main.yml` is automatically loaded and made available in a role.

Reply:
Done.